mkdocs.yml

```
@@ -208,8 +208,35 @@ plugins:
             - url: https://typing-extensions.readthedocs.io/en/latest/objects.inv
             - url: https://rich.readthedocs.io/en/stable/objects.inv
             # waiting for https://github.com/encode/httpx/discussions/3091#discussioncomment-11205594
+  - llmstxt:
+      enabled: !ENV [CI, false]
+      full_output: llms-full.txt
+      markdown_description: |-
+        PydanticAI is a Python agent framework designed to make it less painful to build production grade
+        applications with Generative AI.
+      sections:
+        Concepts documentation:
+          - agents.md
+          - models.md
```

I'm merging a PR that adds a directory `models/`.

```suggestion
          - models/*.md
```